report_backend/time_spent_report.py

```diff
@@ -0,0 +1,138 @@
+"""
+Time spent report backend.
+"""
+import csv
+from datetime import datetime
+import os
+
+import boto3
+
+
+from .base import AbstractBaseReportBackend
+
+
+class TimeSpentReportBackend(AbstractBaseReportBackend):
+    """
+    Backend for time spent report.
+    """
+
+    def __init__(self, **kwargs):
+        super(TimeSpentReportBackend, self).__init__(**kwargs)
+
+
+    def json_report_to_csv(self, json_report_data):
+        """
+        Process json data to convert into csv format.
+
+        Args:
+            json_report_data: Json representacion containing time spent report data.
+        """
+        report_data = json_report_data.get('result', {})
+
+        if not report_data:
+            print('No report data...')
+            exit()
+
+        if report_data.get('time_spent_data'):
+            time_spent_data = report_data.get('time_spent_data')
+            for course in time_spent_data.keys():
```

Comment:
could you iterate over the items and get key and value in order to replace time_spent_data.get(course)

Reply:
Great. @andrey-canon